samples/net/zperf/src/zperf_udp_uploader.c

```diff
@@ -323,9 +323,13 @@ void zperf_udp_upload(const struct shell *shell,
 		}
 
 		/* Wait */
+#if defined(CONFIG_ARCH_POSIX)
+		k_busy_wait(K_MSEC(100));
```

Comment:
1. Why don't use k_sleep() for such a long delay?
2. Why code in "#else" delays for value of `delay` var, while here 100ms is hardcoded?

Reply:
>     1. Why don't use k_sleep() for such a long delay?

See https://docs.zephyrproject.org/latest/boards/posix/native_posix/doc/index.html?highlight=native_posix#working-around-these-limitations for rationale